### ISSUE TITLE
Mutation with directive: CREATE becomes MATCH

### ISSUE THREAD

**Issue by bebbi** [neo4j-graphql/neo4j-graphql-js#35]
With this mutation in the schema:

```
type Mutation {
  createLinkedItem(id: ID!, node: String!, prevId: ID!): Item
    @cypher(statement: "CREATE (n {id: $id, node: $node, createdAt: timestamp()}) WITH n MATCH (prev {id: $prevId}) WITH n, prev MERGE (prev)<-[:LINKS]-(n) RETURN n")
}
```
in debug mode, I can the following console output on the query sent to the db:
```
MATCH (item:Item {id:"xedlgd",node:"myUserString",prevId:"myPrevNode"}) RETURN item { .id , .node , .createdAt ,links: [ item_links IN apoc.cypher.runFirstColumn("MATCH (this)-[:LINKS]->(n) RETURN (n)", {this: item}, true) | item_links { .id }] } AS item SKIP 0
```

Related to #22?
Is there a known workarounds for this?

**Comment by fatjoni** [comment]
I am experiencing the same issue with the latest version

**Comment by bebbi** [comment]
Essentially, I believe the point that could be documented more prominently is that the library currently does not support any top level cypher directive, whether it is mutation or query fields.

After some more playing, I find that this limitation, while not ideal, is not blocking for queries, as in the situations I have encountered I can use a default query as an entrypoint and use the graphql way of exploring the graph with sub-fields and adding custom cypher directives there.

I'll add some thought on how we handle mutations in #9.

Closing as a duplicate of #22 or #9.
